### ISSUE TITLE
Error 1067 when running prepare db

### ISSUE THREAD

**Issue by znb** [kolide/fleet#1620]
### What version of `fleet` are you using (`fleet version --full`)?
```
fleet - version 1.0.5
  branch: 	HEAD
  revision: 	baeb80f3b58f4c8c5c92f94f935674d3926907e1
  build date: 	2017-10-18T16:22:07Z
  build user: 	marpaia
  go version: 	go1.9.1
```

### What operating system are you using?
Ubuntu 16.04

### What did you do?
```
 ./fleet_linux_amd64 prepare db  --mysql_address=127.0.0.1:3306     --mysql_database=kolide     --mysql_username=kolideuser --mysql_password=happy123
```

### What did you expect to see?
The create DB output 

### What did you see instead?
```
2017/11/15 18:42:06 FAIL 20161118212641_CreateTablePasswordResetRequests.go (Error 1067: Invalid default value for 'expires_at'), quitting migration.
```

**Comment by groob** [comment]
Use mysql 5.7 (see docs for requirements)

**Comment by marpaia** [comment]
I'm going to close this as I also recall seeing this issue solved by using MySQL 5.7. Please re-open if the problem persists after upgrading MySQL though.

**Comment by znb** [comment]
Thank you all.

I'm using MySQL 5.7.20-0ubuntu0.16.04.1
